pywps/config.py

```diff
@@ -75,7 +80,28 @@ def loadConfiguration(cfgfiles=None):
 
     config = ConfigParser.ConfigParser()
     config.read(cfgfiles)
-   
+
+    # adapting config from env_vars
+    # each env_var following the format PYWPS_CONFIG_section_option is added to the config
+    _overrideConfigFromConfigDict({k.split("_",2)[-1]:v for k,v in os.environ.iteritems() if k.startswith('PYWPS_CONFIG_')})
+
+    # same trick for request environ, overriding previous config settings 
+    if environ:
+        _overrideConfigFromConfigDict({k.split("_",2)[-1]:v for k,v in environ.iteritems() if k.startswith('PYWPS_CONFIG_')})
+
+def _overrideConfigFromConfigDict(env_config):
+    # In linux we can have camel case env_vars. In windows we'll run into some trouble.
+    # for now, for windows we'll force section and option to lowercase
+    for k,v in env_config.iteritems():
+        section, option = k.split("_",1)
+        try:
+            if sys.platform == 'win32':
+                config.set(section.lower(), option.lower(), v)
```

Comment:
Maybe converting to lower case is not necessary for config sections and options coming from wsgi request environment?